ossapi/enums.py

```diff
@@ -173,6 +173,9 @@ class UserBeatmapType(Enum):
     RANKED_AND_APPROVED = "ranked_and_approved"
     UNRANKED = "unranked"
 
+class SortOrder(Enum):
```

Comment:
there are (going to be be) lots of difference sort enums so this is ambiguous. Should probably be `BeatmapsetDiscussionPostSort` (a mouthful, but not much we can do about that)